### PR TITLE
Fix: input bar hidding long answer in a conversation

### PR DESCRIPTION
**Pull request by lasryaric**
fixes: https://github.com/dust-tt/tasks/issues/197

### CODE REVIEW

File: front/components/assistant/conversation/Conversation.tsx [hunk 1]
```diff
@@ -161,7 +161,7 @@ export default function Conversation({
   }
 
   return (
-    <div className="pb-32">
+    <div className="pb-72">
```
Comment (lasryaric) [inline front/components/assistant/conversation/Conversation.tsx:164]: This is pb-72 because the input bar has a m-h-64. pb-64 is not enough.

Comment (PopDaph) [reply front/components/assistant/conversation/Conversation.tsx:164]: That means we have a big blank when the input bar is not big. How we sure we want that? 🙏🏻

Comment (spolu) [reply front/components/assistant/conversation/Conversation.tsx:164]: Did you check how it impacts auto-scrolling for long conversations?

There's also a world in which we don't fix this. On this one the boundary between bug and feature is quite blurry

**Comment by lasryaric** [comment]
Closing this PR without merging.
The scope of this task is bigger than the eng-runner scope.

**Pull request closed without merging**